itests/kit/node_unmanaged.go

```diff
@@ -788,6 +783,20 @@ func (tm *TestUnmanagedMiner) calculateNextPostEpoch(
 
 	tm.t.Logf("Miner %s: WindowPoST(%d): ProvingDeadline: %+v", tm.ActorAddr, sectorNumber, di)
 
+	if di.Open > di.CurrentEpoch {
```

Comment:
Just because deadline open is > current epoch, how can we conclude that this is a new sector yet to be ticked by cron ?  This could also be the case for sector that have already been ticked by cron right ?

Reply:
@rvagg Honestly, I think it'd be much easier to reason about this if we wait for one challenge epoch to pass after onboarding a sector and get the new values.

Reply:
no, this is the miner's deadline info not sector info, I _think_ this only happens upon first enrol